site/_docs/provisioning/adop_cli.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: docs
+chapter: Provisioning
+title: Using the adop cli 
+permalink: /docs/provisioning/adop_cli/
+---
+
+# ADOP CLI Instructions
+
+These instructions will spin up an instance in a single server after it has been provisioned by using the ADOP CLI.  
+
+Please check the [prerequisites](http://accenture.github.io/adop-docker-compose/docs/prerequisites/).
+
+1. Clone the [ADOP git repository](https://github.com/Accenture/adop-docker-compose) and cd into it.
+2. Run the ADOP CLI with the added parameter for the Proxy IP.
+  1. This IP must be reachable otherwise the startup will hang.
```

Comment:
This is not intended properly, coming up as "3", which conflicts with the 3 on the lines below.
![pr_175-adop_cli md](https://user-images.githubusercontent.com/10450980/32235704-1ee88442-be58-11e7-8058-77af2e39a3e5.jpeg)

Reply:
fixed in the latest commit